src/tasks/AotCompilerTask/MonoAOTCompiler.props

```diff
@@ -14,7 +14,10 @@
     <MonoAOTCompilerDefaultProcessArguments Include="-O=gsharedvt" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetOS)' == 'Android'">
-    <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'arm'" Include="mtriple=armv7-linux-gnueabi-thumbv7s" />
+    <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'arm'" Include="mtriple=armv7-linux-gnueabi" />
```

Comment:
this should be `armv7-linux-androideabi`

Reply:
I'll try it.  Zoltan gave me suggestions to try ;-)

Reply:
The aot compiler uses -gnueabi to enable some features, so either it needs to stay, or the aot compiler has to be modified to recognize androideabi as well.